HARDeConstruction/FounderMatching
# [FEAT] FND-32: Create Next.js Application

This PR implements the initial setup of a Next.js application. 

Changes include:
- Removed unnecessary files from the repository to prepare for the Next.js setup.
- Created a new Next.js app in the root folder.

Testing Steps:
- Make sure the Next.js app runs successfully using `npm run dev`.
- Verify the application renders the default Next.js page without issues.
